doc/index.rst

```diff
@@ -721,6 +721,37 @@ Functions
 
    .. versionadded:: 4.1.0
 
+
+Helper classes
+~~~~~~~~~~~~~~
+
+.. class:: Doc(/, documentation)
```

Comment:
I'm not sure this is the right syntax in Sphinx to show that it's a positional-only parameter, I assumed so infering from `dataclass_transform(*, eq_default=False, etc=True)`.

Reply:
This was already wrong, just updated it and added the comment again in the new version.